session_security/static/session_security/script.js

```diff
@@ -66,7 +66,12 @@ yourlabs.SessionSecurity.prototype = {
 
     // Called by click, scroll, mousemove, keyup.
     activity: function() {
-        this.lastActivity = new Date();
+        var now = new Date();
+        if (now - this.lastActivity < 1000)
```

Comment:
Very nice explanation, could you refer to this pull request in the commit message please so that others could find it ? Then I think we're good to merge and release B)